### DISCUSSION TITLE
Own Exams

### DISCUSSION THREAD

**Discussion by MartinHin** [SafeExamBrowser/seb-server#77] [Ideas]
Hello
We are using the Safe Exam Browser for "Check-Dein-Wissen". With Microsoft Forms, we are using it more and more for our own Tests.
I am missing my Part to add a Exam without a LMS. Or did i miss something?
We would love the Possibility to add our Exam Config just as own Exam without any Moodle oder other LMS Config, so our Students can choose the correct Exam when starting the Safe Exam Browser(like the Demo ones).
Thank you!

**Comment by anhefti** [comment]
Hi,

Since security is an integral part of SEB Server, LMS binding is mandatory for setup and monitor a secure exam with SEB and SEB Server.
Please also see discussion: https://github.com/SafeExamBrowser/seb-server/discussions/47

If you want to just setup an exam with SEB and no LMS but with some alternative webpage, you can do so by using the SEB configuration tool to create SEB configurations with appropriate start URL.

Best regards

**Reply by MartinHin** [reply]
i see.
very sad.
as long as you would just provide the possibility to configure our own exam, it would be usable for our schools in a easy way. i mean, the possibility to configure other exam than LMS based ones, does not mean that all have to use this, but Sekundar, Real und Bezirksschulen would be able to configure it with the mandatory exams the have to fulfill.
and no, i am not a teacher, just a it guy who tries to make it easier for our teachers.
also best regards from zofingen

**Reply by danschlet** [reply]
We may discuss in our team how to enable easier integration with SEB Server for currently unsupported web based systems used for e-assessment in the future.

Are you aware that setting up, running and maintaining an own SEB Server instance is quite effortful? Check-Dein-Wissen for example currently also doesn't use SEB Server. It surely makes sense to use SEB Server in future, but you'll only really benefit from the extra effort if there is an integration with the e-assessment system you're using. For example you can only see the user name of an exam participant in SEB Server if the e-assessment system/LMS has a SEB Server integration. 

If you're looking for an easy way how to use your own exams with SEB, then we still would recommend to create simple SEB configuration files in the SEB Configuration Tool or the Preferences editor in the SEB macOS version. You can use default settings, just set your own SEB admin and quit passwords and the according start URL. Then save the .seb config file and place it onto a webserver and create a link to it using sebs:// links on a simple HTML portal page (like for example https://safeexambrowser.org/exams). You can also do that in a CMS (content management system) you might be using. Also the next SEB releases will support creating sebs:// links containing a full SEB configuration, so you won't need to save and upload .seb config files separately.

**Comment by MartinHin** [comment]
Guten Tag Daniel

Entschuldigung, aber mein Englisch ist nur bedingt
Konversationstauglich. 

Ich erlaube mir das Du, falls das nicht in Ordnung ist, bitte ich um
Korrektur. 

Als kurze Erklärung:
Ich arbeite bei der Kreisschule aargauSüd als Hausinterner Informatiker.
Das bedeutet, dass ich mehr die technischen Probleme als die
Pädagogischen sehe.
Bei uns ist es so, dass vorwiegend Jugendliche und noch nicht Erwachsene
die für die Schule verpflichtenden Prüfungen absolvieren müssen. 

Unser grösstes Problem ist, dass Lehrpersonen die Quit Kennwörter auf
Zettel schreiben, diese dann am Monitor ankleben und die SuS
(Schülerinnen und Schüler) diese dann finden.
Oder wenn die Kennwörter nicht öffentlich gemacht werden (nach der
Prüfung) dann muss die Lehrperson von Laptop zu Laptop gehen und diese
eintippen, damit die SuS wieder aus dem SEB aussteigen können. 

Für uns wäre es deshalb eine unglaublich edle Lösung, wenn wir die EXAM
Configurations per SEB Server direkt als EXAM publizieren könnten. Die
SuS würden beim starten des SEB eine Liste der verfügbaren Prüfungen
sehen: Beispiel "Check Dein Wissen 1", "R1a Mathematik", usw. diesen
dann lösen und die Lehrperson kann nachher per SEB Server die SEB
schliessen, bzw. freigeben. 

Ich weiss, dass das nicht die Art der Verwendung ist, die Ihr für den
SEB Server dachtet, aber vielleicht wäre es eine tolle Art dies für
andere Schulen nutzbar zu machen. 

Sollte dies keine Option sein/werden, dann werden wir natürlich
weiterhin mit der Examenkonfiguration arbeiten. 

Danke für Deine Zeit, beste Grüsse aus Reinach AG 

Martin Hinnen 

Am 2023-05-12 11:30, schrieb Daniel R. Schneider:

> We may discuss in our team how to enable easier integration with SEB Server for currently unsupported web based systems used for e-assessment. 
> 
> Are you aware that setting up, running and maintaining an own SEB Server instance is quite effortful? Check-Dein-Wissen for example currently also doesn't use SEB Server. It surely makes sense to use SEB Server in future, but you'll only really benefit from the extra effort if there is an integration with the e-assessment system you're using. For example you can only see the user name of an exam participant in SEB Server if the e-assessment system/LMS has a SEB Server integration. 
> 
> If you're looking for an easy way how to use your own exams with SEB, then we still would recommend to create simple SEB configuration files in the SEB Configuration Tool or the Preferences editor in the SEB macOS version. You can use default settings, just set your own SEB admin and quit passwords and the according start URL. Then save the .seb config file and place it onto a webserver and create a link to it using sebs:// links on a simple HTML portal page (like for example https://safeexambrowser.org/exams). You can also do that in a CMS (content management system) you might be using. Also the next SEB releases will support creating sebs:// links containing a full SEB configuration, so you won't need to save and upload .seb config files separately. 
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
> You are receiving this because you authored the thread.Message ID: ***@***.***>
 

Links:
------
[1]
https://github.com/SafeExamBrowser/seb-server/discussions/77#discussioncomment-5882926
[2]

**Reply by danschlet** [reply]
Hallo Martin

Wir haben uns die Sache mit SEB Server Einsatz ohne Integration im Prüfungssystem angeschaut und werden dies weiterverfolgen, sodass es dafür Unterstützung geben wird in zukünftigen SEB Server und SEB Client Versionen.

Eine andere Frage: Wir wurden mehrfach darauf hingewiesen, dass Microsoft Forms für Prüfungen eingesetzt wird, aber wir haben das nie selber gesehen. Könntest du uns eventuell einen Link auf eine simple Beispielprüfung senden, sodass wir uns ein Bild davon machen können?

Danke und beste Grüsse
Daniel

**Comment by MartinHin** [comment]
Grüss Dich Daniel 

Vielen Dank für das, für sehr positive, Feedback! 

Ich frage mal ein paar unserer Lehrpersonen betreffend dieser "Forms
Prüfungen".
Gerne sende ich Dir dann ein paar Links. 

Beste Grüsse, Martin 

Am 2023-06-26 09:42, schrieb Daniel R. Schneider:

> Hallo Martin 
> 
> Wir haben uns die Sache mit SEB Server Einsatz ohne Integration im Prüfungssystem angeschaut und werden dies weiterverfolgen, sodass es dafür Unterstützung geben wird in zukünftigen SEB Server und SEB Client Versionen. 
> 
> Eine andere Frage: Wir wurden mehrfach darauf hingewiesen, dass Microsoft Forms für Prüfungen eingesetzt wird, aber wir haben das nie selber gesehen. Könntest du uns eventuell einen Link auf eine simple Beispielprüfung senden, sodass wir uns ein Bild davon machen können? 
> 
> Danke und beste Grüsse
> Daniel 
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
> You are receiving this because you authored the thread.Message ID: ***@***.***>
 

Links:
------
[1]
https://github.com/SafeExamBrowser/seb-server/discussions/77#discussioncomment-6278969
[2]
https://github.com/notifications/unsubscribe-auth/A7YSNSAKX4QXCVVMZFCNZZTXNE4NFANCNFSM6AAAAAAX6JD3TU
--=_db01ac7b4f0845ff7f8d56e98bc9facd
Content-Transfer-Encoding: quoted-printable
Content-Type: text/html; charset=UTF-8

<html><head><meta http-equiv=3D"Content-Type" content=3D"text/html; charset=
=3DUTF-8" /></head><body style=3D'font-size: 10pt; font-family: Verdana,Gen=
eva,sans-serif'>
<p>Gr&uuml;ss Dich Daniel</p>
<p>Vielen Dank f&uuml;r das, f&uuml;r sehr positive, Feedback!</p>
<p>Ich frage mal ein paar unserer Lehrpersonen betreffend dieser "Forms Pr&=
uuml;fungen".<br />Gerne sende ich Dir dann ein paar Links.</p>
<p>Beste Gr&uuml;sse, Martin</p>
<p><br /></p>
<div id=3D"signature">&nbsp;</div>
<p><br /></p>
<p id=3D"reply-intro">Am 2023-06-26 09:42, schrieb Daniel R. Schneider:</p>
<blockquote type=3D"cite" style=3D"padding: 0 0.4em; border-left: #1010ff 2=
px solid; margin: 0">
<div id=3D"replybody1">
<p><br /></p>
<p dir=3D"auto">Hallo Martin</p>
<p dir=3D"auto">Wir haben uns die Sache mit SEB Server Einsatz ohne Integra=
tion im Pr&uuml;fungssystem angeschaut und werden dies weiterverfolgen, sod=
ass es daf&uuml;r Unterst&uuml;tzung geben wird in zuk&uuml;nftigen SEB Ser=
ver und SEB Client Versionen.</p>
<p dir=3D"auto">Eine andere Frage: Wir wurden mehrfach darauf hingewiesen, =
dass Microsoft Forms f&uuml;r Pr&uuml;fungen eingesetzt wird, aber wir habe=
n das nie selber gesehen. K&ouml;nntest du uns eventuell einen Link auf ein=
e simple Beispielpr&uuml;fung senden, sodass wir uns ein Bild davon machen =
k&ouml;nnen?</p>
<p dir=3D"auto">Danke und beste Gr&uuml;sse<br />Daniel</p>
<p style=3D"font-size: small; -webkit-text-size-adjust: none; color: #666;"=
>&mdash;<br />Reply to this email directly, <a href=3D"https://github.com/S=
afeExamBrowser/seb-server/discussions/77#discussioncomment-6278969" target=
=3D"_blank" rel=3D"noopener noreferrer">view it on GitHub</a>, or <a href=
=3D"https://github.com/notifications/unsubscribe-auth/A7YSNSAKX4QXCVVMZFCNZ=
ZTXNE4NFANCNFSM6AAAAAAX6JD3TU" target=3D"_blank" rel=3D"noopener noreferrer=
">unsubscribe</a>.<br />You are receiving this because you authored the thr=
ead.<img src=3D"https://github.com/notifications/beacon/A7YSNSGCCMDIHV2HOCO=
GO2TXNE4NFA5CNFSM6AAAAAAX6JD3TWWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23=
LFNZ2KUY3PNVWWK3TUL5UWJTQAL7HTS.gif" width=3D"1" height=3D"1" /><span style=
=3D"color: transparent; font-size: 0; display: none; visibility: hidden; ov=
erflow: hidden; opacity: 0; width: 0px; height: 0px; max-width: 0; max-heig=
ht: 0; mso-hide: all;">Message ID: <span>&lt;SafeExamBrowser/seb-server/rep=
o-discussions/77/comments/6278969</span><span>@</span><span>github</span><s=
pan>.</span><span>com&gt;</span></span></p>
</div>
</blockquote>
</body></html>

--=_db01ac7b4f0845ff7f8d56e98bc9facd--
